frontend/src/components/Notification.tsx

```diff
@@ -0,0 +1,55 @@
+import { Snackbar, makeStyles } from '@material-ui/core'
+import * as React from 'react';
+import Alert from '@material-ui/lab/Alert';
+
+
+export type notificationType =  {
+    isOpen: boolean;
+    type: 'error' | 'info' | 'success' | 'warning'; 
+    message: string;
+};
+
+
+
+const useStyles = makeStyles(theme => ({
+    root:{
+        top: theme.spacing(9) //Change this depending on size of header
+    }
+}));
+
+export default function Notification(props) {
+    const [notify, setNotify] = React.useState<notificationType>(props);
+    const classes = useStyles()
+
+    React.useEffect(() => {
+        setNotify(props);
+      }, [props]);
```

Comment:
I don't understand this pattern of mirroring the props in the state. Why are we doing this? What is the purpose?